src/stories/Interactive/Code/CodeBlock.stories.tsx

```diff
@@ -55,3 +59,42 @@ InsideAccordionWithFileName.args = {
   filename: 'Example File Name',
   filenameColor: '#00ff00',
 };
+
+/*
+ * See https://storybook.js.org/docs/react/writing-stories/play-function#working-with-the-canvas
+ * to learn more about using the canvasElement to query the DOM.
+ */
```

Comment:
We might want to create a `CONTRIBUTING.md` and put this information there. We should do this in a future PR.

Reply:
Because we'd want to continue to write thorough tests for our UI and it probably isn't productive to copy paste this everywhere we want to create interactions on a storybook story

Reply:
I added some parts to Contributing [here](https://github.com/mintlify/components/pull/72).

We could add what is specific to `components` in the `mintlify/components` [Contributing wiki page](https://github.com/mintlify/components/wiki) or create an additonal `CONTRIBUTING.md`, link to it from `README.md` and move the `Contributing` parts:

![image](https://user-images.githubusercontent.com/34443492/220265262-b54d6e35-3bdb-4031-ab11-81ab2a3811c5.png)